### PR TITLE
Adding `code` property to `SyntheticKeyboardEvent` with test

### PR DESCRIPTION
**Pull request by flaviolivolsi**
This is about feature request #11306

### CODE REVIEW

File: packages/react-dom/src/events/SyntheticKeyboardEvent.js [hunk 1]
```diff
@@ -54,6 +54,23 @@ var KeyboardEventInterface = {
...
+
+    // It returns a value not altered by keyboard layout
+    // or the state of the modifier keys.
+
+    // Assuming the input doesn't come from a physical keyboard, but instead from a virtual
+    // keyboard or accessibility device, it will be returned a value set by the browser to match
+    // as closely as possible to what would happen with a physical keyboard, to maximize
+    // compatibility between physical and virtual input devices.
+    if (event.type === 'keydown' || event.type === 'keyup') {
+      return event.code;
```
Comment (aweary) [inline packages/react-dom/src/events/SyntheticKeyboardEvent.js]: According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code), `code` isn't widely supported. Is there a way to cheaply polyfill this for those browsers?

**Comment by flaviolivolsi** [comment]
[This](https://github.com/inexorabletash/polyfill/blob/master/keyboard.js) is one polyfill I could find. Alas, nothing much else.

**Comment by syranide** [comment]
I'm assuming it would be a good idea to polyfill the ones we can universally detect as well, like we do for `key`. The list of possible keys should probably be more or less the same too.

**Comment by flaviolivolsi** [comment]
Is there any guideline for including polyfills?

**Comment by gaearon** [comment]
Closing as stale.

**Pull request closed without merging**